templates/joomla/html/com_content/article/requirements.php

```diff
@@ -133,6 +133,80 @@
 		<div class="alert alert-info"><?php echo JText::sprintf('TECHNICAL_REQUIREMENTS_SUPPORTED_BROWSERS', JHtml::_('link', 'https://docs.joomla.org/S:MyLanguage/Joomla_Browser_Support', 'https://docs.joomla.org/Joomla_Browser_Support', ['class' => 'alert-link'])); ?></div>
 
 		<h2><?php echo JText::_('TECHNICAL_REQUIREMENTS_HEADING_REQUIREMENTS_FOR_UNSUPPORTED_SOFTWARE'); ?></h2>
+        <h3><?php echo JText::_('TECHNICAL_REQUIREMENTS_HEADING_REQUIREMENTS_FOR_JOOMLA_4'); ?></h3>
+        <div class="alert alert-warning"><?php echo JText::_('TECHNICAL_REQUIREMENTS_ALERT_UNRELEASED_VERSION'); ?></div>
+        <table class="table table-bordered table-striped">
+            <thead>
+            <tr>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SOFTWARE'); ?></th>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_RECOMMENDED'); ?></th>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_MINIMUM'); ?></th>
```

Comment:
```suggestion
                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_MINIMUM'); ?><sup><a href="#footnote-minimum">[7]</a></th>
```

And add a footnote with some text that "Minimum requirements is the minimum supported version and versions may or may not work"

Reply:
If it's a minimum supported verison we kinda guarentee it to work. Either way that's not j4 specific. That's what our recommended version is based around no?

Reply:
You remember the discussion about the motion text after it got public ? It was wrongly interpreted as minimum required version and not as minimum supported version. Which could increase our support load .